CMSgov/mint-app
# [EASI-2901] Modular GQL schema support

# EASI-2901


## Changes and Description

- Update linter
- Update gqlgen
- Update Apollo configs
- GQL Extraction
  - Plan Basics
  - Plan Ops Eval and Learning
  - Scalars
  - Shared Enums
  - Directives



## How to test this change

- Build and run the application as normal.
- Smoke test endpoints, especially regarding Plan Basics and Plan Ops Eval and Learning.
- Assert that GQL generation works as expected
  - Extracted resolvers are now placed into their own resolver files based on GQL module names, E.G. `plan_basics.resolvers.go`

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
